api/controllers/UserController.js

```diff
@@ -331,7 +331,7 @@ module.exports = {
           res.json('ok', 200);
         }, function(error) {
           console.log(error);
-          res.json({error: error}, 500);
+          res.json(error, 500);
         });
       });
     });
```

Comment:
Fix for an unrelated issue where the ban button was giving non-useful error messages.